synapse/lib/stormlib/aha.py

```diff
@@ -49,13 +52,71 @@ class AhaLib(s_stormtypes.Lib):
          'type': {'type': 'function', '_funcname': '_methAhaList', 'args': (),
                   'returns': {'name': 'Yields', 'type': 'list',
                               'desc': 'The AHA service dictionaries.', }}},
+        {'name': 'callPeerApi', 'desc': '''Call an API on all peers (leader and mirrors) of an AHA service and yield the responses from each.
+
+        Examples:
+            Call getCellInfo on an AHA service::
+
+                $todo = $lib.utils.todo('getCellInfo')
+                for $info in $lib.aha.callPeerApi(cortex..., $todo) {
+                    $lib.print($info)
+                }
+
+            Call method with arguments::
+
+                $todo = $lib.utils.todo(('method', (1, 2), ({'foo': 'bar'})))
+                for $info in $lib.aha.callPeerApi(cortex..., $todo) {
+                    $lib.print($info)
+                }
+
+        ''',
+         'type': {'type': 'function', '_funcname': '_methCallPeerApi',
+                  'args': (
+                      {'name': 'svcname', 'type': 'str',
+                       'desc': 'The name of the AHA service to call. It is easiest to use the relative name of a service, ending with "...".', },
+                      {'name': 'apiname', 'type': ('str', 'tuple'),
+                       'desc': 'The name of the API to call or a todo tuple (name, args, kwargs).'},
+                      {'name': 'timeout', 'type': 'int', 'default': None,
+                       'desc': 'Optional timeout in seconds.'},
+                      {'name': 'skiprun', 'type': 'str', 'default': None,
+                       'desc': 'Optional run ID argument allows skipping self-enumeration when the caller provides its own run ID.'},
```

Comment:
How do I know what my run ID is?

Reply:
Not sure how to obtain it directly and it isn't used for the mirror tool per-say. It was plumbed in based on the Gather PR.

Reply:
Maybe we don't need it exposed then?

Reply:
@invisig0th says you can get your runid from here: `skiprun=$lib.cortex.getCellInfo().cell.run`. Can you add this info into the description for these two APIs? And maybe add a test to make sure it works.

Reply:
next commit will have more description and tests for runid.